src/IECoreArnold/ShaderNetworkAlgo.cpp

```diff
@@ -1003,14 +1037,6 @@ void IECoreArnold::ShaderNetworkAlgo::convertUSDShaders( ShaderNetwork *shaderNe
 		{
 			newShader = new Shader( "quad_light", "ai:light" );
 			transferUSDLightParameters( shaderNetwork, handle, shader.get(), newShader.get() );
-			const float width = parameterValue( shader.get(), g_widthParameter, 1.0f );
```

Comment:
We need to transfer the width and height to `newShader` using `transferUSDParameter()`, bearing in mind that the USD defaults are different to the Arnold ones (would be good to add test coverage for this case). If we do need to change the vertex order though, we need to be wary of how this impacts the orientation of any textures applied to the light - it's possible that we actually want to keep on defining vertices here, if the USD ones require a different order to the default Arnold ones.

Reply:
> it's possible that we actually want to keep on defining vertices here

Yes, I think we do need to keep defining the vertices for USD lights. To get the proper orientation for what I can gather from the spec is the right layout we need to start from a different corner point when converting to Arnold, and add a `visualiserOrientation` for GL.

Those are both done in https://github.com/GafferHQ/gaffer/pull/5169/commits/7b28509c04775978ef866e9255b7ce3d92fbb516

Reply:
Did you manage to get a practical reference for a textured USD quad light? If not, maybe we can get one out of Houdini, @murraystevenson?